test/units/bodyParser.formData.test.ts

```diff
@@ -0,0 +1,47 @@
+import { formData } from "../../mod.ts";
+import { expect } from "../deps.ts";
+import { describe, it } from "../utils.ts";
+
+const mockFormData = new FormData();
+
+mockFormData.append("hello", "deno");
+
+const textHeaders = new Headers();
+textHeaders.set("Content-Type", "text/plain");
+textHeaders.set("Content-Length", "1");
+
+const formDataHeaders = new Headers();
+formDataHeaders.set(
+  "Content-Type",
+  "multipart/form-data; boundary=----opineBoundary",
+);
```

Comment:
It would be good to have at least one non-trivial test case for a request with form data

Reply:
what do you mean by this? do you mean unsual content-types and also missing one?

Reply:
found way more issues on the way there, but have finished adding more tests